src/components/Shared/SpectralSettings/SpectralSettingsComponent.tsx

```diff
@@ -39,6 +42,17 @@ export class SpectralSettingsComponent extends React.Component<{
                         onChange={(event: React.FormEvent<HTMLSelectElement>) => this.props.onSpectralCoordinateChange(event.currentTarget.value as string)}
                     />
                 </FormGroup>
+                {this.props.secondaryAxisCursorInfoVisible && (
+                    <FormGroup label={"Secondary Coordinate"} inline={true} disabled={disableCoordinateSetting}>
+                        <HTMLSelect
+                            disabled={disableCoordinateSetting}
+                            value={frame && frame.spectralCoordinateSecondary ? frame.spectralCoordinateSecondary : ""}
+                            options={spectralCoordinateOptions}
+                            onChange={(event: React.FormEvent<HTMLSelectElement>) => this.props.onSpectralCoordinateChangeSecondary(event.currentTarget.value as string)}
```

Comment:
can use implicit typing here (`{event=>this.props.onSpectral...}`)

Reply:
applied implicit typing